@xen-orchestra/lite/src/libs/xen-api/xen-api.types.ts

```diff
@@ -60,6 +60,7 @@ type ObjectTypeToRecordMapping = {
   message: XenApiMessage<any>
   network: XenApiNetwork
   pbd: XenApiPbd
```

Comment:
```suggestion
  pbd: XenApiPbd
  pci: XenApiPci
```

Don't forget to add the store :grin: